jade/hpc/slurm_manager.py

```diff
@@ -204,11 +204,20 @@ def get_num_cpus():
 
     def list_active_nodes(self, job_id):
         out1 = {}
-        check_run_command(f'squeue -j {job_id} --format="%500N" -h', out1)
-        nodes_compact = out1["stdout"].strip()
+        # It's possible that 500 characters won't be enough, even with the compact format.
```

Comment:
This is really an addendum to the multi-node PR. It adds a check to ensure the result is consistent. I tested on the HPC this morning. We should probably make a standard way to run _short_ unit tests on Eagle.